### PR TITLE
Update Russian translation

### PR DESCRIPTION
**Pull request by adem4ik**

### CODE REVIEW

File: avidemux_ru.ts [hunk 1]
```diff
@@ -8417,7 +8418,7 @@ Start and/or end time would be out of range</source>
     <message>
         <location line="-3"/>
         <source>Main10</source>
-        <translation type="unfinished"></translation>
+        <translation>Основной10</translation>
```
Comment (eumagga0x2a) [inline avidemux_ru.ts]: If you feel that the profile name needs translation, I would suggest writing the profile name with a space. "Main10" in the source is apparently a mixup of "Main 10" and "main10" (the command line parameter) and needs to be fixed.

File: avidemux_ru.ts [hunk 2]
```diff
@@ -9395,7 +9396,7 @@ I will assume it has and try to continue</source>
         <location line="+3"/>
         <location filename="../../../avidemux_plugins/ADM_videoFilters6/ivtcDupeRemover/ADM_ivtcDupeRemover.cpp" line="+3"/>
         <source>_Frame rate change:</source>
-        <translation>_Изменение частоты кадров:</translation>
+        <translation>_Смена кадровой частоты:</translation>
```
Comment (eumagga0x2a) [inline avidemux_ru.ts]: Are you sure? This feels really wrong for me (and is very unpopular according to $SEARCH_ENGINE).

File: avidemux_ru.ts [hunk 3]
```diff
@@ -10595,22 +10596,22 @@ Only VP8/VP9/AV1 video and Vorbis/Opus audio supported</source>
...
         <location line="+12"/>
         <source>Only MPEG-4, H264, H265, AV1, DNxHD and ProRes supported for video</source>
-        <translation>Только MPEG-4, H264, H265, AV1, DNxHD и ProRes поддерживаются как видео</translation>
+        <translation>Для видео поддерживаются только MPEG-4, H264, H265, AV1, DNxHD и ProRes</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Only MPEG-1/2/4, H264, H265 and AV1 supported for video</source>
-        <translation>Только MPEG-1/2/4, H264, H265 и AV1 поддерживаются как видео</translation>
+        <translation>Для видео поддерживаются только  MPEG-1/2/4, H264, H265 и AV1</translation>
```
Comment (eumagga0x2a) [inline avidemux_ru.ts]: A tiny nit: a superfluous space between "только" and "MPEG-1/2/4".

File: avidemux_ru.ts [hunk 4]
```diff
@@ -13890,7 +13891,8 @@ Only VP8/VP9 video and Vorbis/Opus audio supported</source>
         <location line="+0"/>
         <source>this mode has not been implemented
 </source>
-        <translation>этот режим ещё не разработан</translation>
+        <translation>этот режим ещё не разработан
+</translation>
```
Comment (eumagga0x2a) [inline avidemux_ru.ts]: Definitely a bug in the source, fixed in the translation, now resurrected.

File: avidemux_ru.ts [hunk 5]
```diff
@@ -11475,17 +11475,17 @@ The file must be re-indexed. Proceed?</source>
...
         <location line="+1"/>
         <source>Recent Projects</source>
-        <translation>Предыдущие проекты</translation>
+        <translation>Недавние проекты</translation>
     </message>
     <message>
         <location line="+32"/>
         <source>Restore defaults</source>
-        <translation>Восстановить стандартный набор</translation>
+        <translation>Сбросить выбор</translation>
```
Comment (eumagga0x2a) [inline avidemux_ru.ts]: The default is everything enabled / selected. The user chooses to _deselect_ an option. I feel a little bit of cognitive dissonance with the (standard) wording of the new translation, would prefer the previous one.

File: avidemux_ru.ts (hunk 1 above)
Comment (adem4ik) [reply avidemux_ru.ts]: Okey, I'll fix it

File: avidemux_ru.ts (hunk 2 above)
Comment (adem4ik) [reply avidemux_ru.ts]: Will be reverted, but it's not wrong, just another wording — https://ru.wikipedia.org/wiki/%D0%9A%D0%B0%D0%B4%D1%80%D0%BE%D0%B2%D0%B0%D1%8F_%D1%87%D0%B0%D1%81%D1%82%D0%BE%D1%82%D0%B0

File: avidemux_ru.ts (hunk 3 above)
Comment (adem4ik) [reply avidemux_ru.ts]: Right, will be fixed.

File: avidemux_ru.ts (hunk 4 above)
Comment (adem4ik) [reply avidemux_ru.ts]: Oh, i'll revert it in translation.

File: avidemux_ru.ts (hunk 5 above)
Comment (adem4ik) [reply avidemux_ru.ts]: Okey, will be reverted

**Comment by eumagga0x2a** [comment]
Thank you very much.

I noticed too late that the audio filters dialog was already using "кадровая частота" as the translation for "frame rate", stopping bitching about it. Of course, both variants are valid.

**Pull request merged**